rscripts/create_spatial_example_data/process_spatial_data.R

```diff
@@ -7,13 +7,29 @@ gfdb <- "cache/GF_nat_reg.gdb"
 
 # Clean up HRU spatial data
 hru_loaded <- read_sf(gfdb, "nhru")
+#there are layers that list point/line errors,
+#not sure if these are useful?
+#line_errors <- read_sf(gfdb, "T_1_LineErrors")
+pryr::object_size(hru_loaded$Shape)
+#shape size is yuge
+#all geometries are simple, but some are invalid
 hru_valid <- st_make_valid(hru_loaded)
 hru <- hru_valid %>% 
   select(hru_id_nat) %>%
   st_sf()
 
 st_agr(hru) <- "constant"
 
+#TODO:
+#intersect HRUs with a state so can work with political
+#boundaries and get a sense of scale
```

Comment:
what do you mean "get a sense of scale"? Just wondering the motivation here

Reply:
I was plotting a subset of hucs at varying levels of simplification, but its hard to tell how far to tell what a useful level of detail is without some kind of geographic references (i.e. something known like state lines)